### PR TITLE
Collect Basic RDP settings

### PR DESCRIPTION
**Pull request by YangLu1031**
Collect basic OS rdp settings according to the [troubleshooting guide](https://cloud.google.com/compute/docs/troubleshooting/troubleshooting-rdp#os_configuration)

1. Created an external script `rdp_status.ps1`
2. Bumped the version to `1.1.0@0`
3. Sample of [rdp_status.txt](https://gist.github.com/YangLu1031/06d17984af0515b995ba6023f67f31ea#file-sample-of-rdp_status-txt)

### CODE REVIEW

File: cli_tools/diagnostics/main_windows.go [hunk 1]
```diff
@@ -183,6 +184,20 @@ func gatherProgramLogs(logs chan logFolder, errs chan error) {
 	logs <- logFolder{"Program", runAll(commands, errs)}
 }
 
+// gatherRDPSettings invokes rdp_status.ps1 to collect rdp settings,
+// return rdp_status.txt file path and errors(if any).
+func gatherRDPSettings(logs chan logFolder, errs chan error) {
+	pwshPath, err := exec.LookPath("powershell")
+	if err != nil {
+		errs <- err
```
Comment (pjh) [inline cli_tools/diagnostics/main_windows.go]: Return here (pwshPath may not be set correctly)?

File: cli_tools/diagnostics/rdp_status.ps1 [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  @{
+    description="Check that Remote Connections are enabled, expect [fDenyTSConnections: 0]:";
+    cmd="Write-Host $RDPServerPath'\fDenyTSConnections: '(Get-ItemProperty -Path '$RDPServerPath' -Name fDenyTSConnections -ErrorAction SilentlyContinue).fDenyTSConnections"
+  },
+  @{
+    description="Ensure that the Windows firewall has Remote Desktop Connections enabled, expect [Enabled:Yes]:";
+    cmd="netsh advfirewall firewall show rule name='Remote Desktop - User Mode (TCP-In)'"
+  },
+  @{
+    description="Check what port number is configured for RDP connections, expect [reasonable number, default: 3389]:";
```
Comment (pjh) [inline cli_tools/diagnostics/rdp_status.ps1]: Remove `reasonable number, `

File: cli_tools/diagnostics/rdp_status.ps1 [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    description="Ensure that connected user account has permissions for remote connections, expect [target local/domain username in resulting list]:";
+    cmd="net localgroup 'Remote Desktop Users'"
+  },
+  @{
+    description="Verify that MTU size is no greater than 1460, expect [MTU <= 1460, Interface:Ethernet]:";
+    cmd="netsh interface ipv4 show subinterfaces"
+  }
+)
+
+Write-Host "If you see any unexpected values, please go to troubleshooting guide:`nhttps://cloud.google.com/compute/docs/troubleshooting/troubleshooting-rdp#os_configuration`n"
```
Comment (pjh) [inline cli_tools/diagnostics/rdp_status.ps1]: Remove the `#os_configuration` target?

**Comment by pjh** [comment]
cc @adjackura

**Comment by YangLu1031** [comment]
@pjh please take another look :)

File: cli_tools/diagnostics/google-compute-engine-diagnostics.goospec [hunk 4]
```diff
@@ -1,20 +1,23 @@
...
-  "version": "1.0.0@1",
+  "version": "1.1.0@0",
   "arch": "x86_64",
   "authors": "Google Inc.",
   "license": "http://www.apache.org/licenses/LICENSE-2.0",
   "description": "Google Compute Engine diagnostics tool",
   "files": {
-    "diagnostics.exe": "<ProgramFiles>/Google/Compute Engine/diagnostics/diagnostics.exe"
+    "diagnostics.exe": "<ProgramFiles>/Google/Compute Engine/diagnostics/diagnostics.exe",
+	  "rdp_status.ps1": "<ProgramFiles>/Google/Compute Engine/diagnostics/rdp_status.ps1"
```
Comment (adjackura) [inline cli_tools/diagnostics/google-compute-engine-diagnostics.goospec]: extra spaces

**Comment by YangLu1031** [comment]
@adjackura  addressed the comments, can you take a look again?

**Pull request merged**